test/fixtures/dialects/sparksql/with_insert_with_statement_a.sql

```diff
@@ -0,0 +1,22 @@
+WITH mycte1 AS (
```

Comment:
I would put all your SQLs into the existing `common_table_expressions.sql` instead of making new files.

Reply:
Oh yes, that would be a good Idea, will move these tests over there